app/src/main/java/com/example/tictactoe/MainActivity.kt

```diff
@@ -10,7 +10,7 @@ class MainActivity : AppCompatActivity(R.layout.activity_main) {
             supportFragmentManager
                 .beginTransaction()
                 .add(R.id.home_screen_container, HomeScreen.newInstance())
-                .commitNow()
+                .commit()
```

Comment:
This should work even with commitNow() also. Please check and revert is possible ?

Reply:
Done